src/main/java/javax/validation/metadata/ContainerDescriptor.java

```diff
@@ -0,0 +1,35 @@
+package javax.validation.metadata;
+
+import java.util.Set;
+
+import javax.validation.Valid;
+
+/**
+ * Represents an element that might be a container, thus allowing container element types constraints.
```

Comment:
types=type

Although I'd prefer "container element constraints" (addressing your question in the other thread), as the constraints apply to a container element. The container element type only is what's carrying the configuration.